lib/plugins/physics.js

```diff
@@ -78,35 +91,50 @@ function inject (bot, { physicsEnabled }) {
   function sendPacketPosition (position, onGround) {
     // sends data, no logic
     const oldPos = new Vec3(lastSent.x, lastSent.y, lastSent.z)
-    lastSent.x = position.x
-    lastSent.y = position.y
-    lastSent.z = position.z
-    lastSent.onGround = onGround
-    bot._client.write('position', lastSent)
+    const packet = {}
```

Comment:
Why change the properties on the packet object when you can just do 
```js
const packet = {
  x: position.x,
  y: position.y
  ...
}
```

Reply:
or you can do 
```js
bot._client.write('position', 
  x: position.x,
  y: position.y
  ...
}
```

Reply:
> Why change the properties on the packet object when you can just do
> 
> ```js
> const packet = {
>   x: position.x,
>   y: position.y
>   ...
> }
> ```

I'll change it to this, because I believe it makes it more readable.